hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/PersistentState.java

```diff
@@ -364,7 +366,7 @@ private void setDataNodes(Map<String, String> dataNodes) {
    */
```

Comment:
I don't see the issue with the compiler outputting the warnings here. I think we should be aware of these warnings.